src/handlers/rentals.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable prefer-const */
+import { log } from '@graphprotocol/graph-ts'
+import { BigInt } from '@graphprotocol/graph-ts'
+import { buildCountForAllRentals, buildCountForRentals } from '../modules/count'
+import { Rental } from '../entities/schema'
+import { OperatorUpdated, RentalStarted, AssetClaimed } from '../entities/Rentals/Rentals'
+
+function buildRentalId(contractAddress: string, tokenId: BigInt, index: BigInt): string {
+  return contractAddress + ':' + tokenId.toString() + ':' + index.toString()
+}
+
+export function handleRentalStart(event: RentalStarted): void {
+  let contractAddress = event.params._contractAddress.toHexString()
+  let tokenId = event.params._tokenId
+
+  let rentalIndex = buildCountForRentals(contractAddress, tokenId)
```

Comment:
buildCountForRental might be a better name for the function don't you think?

Reply:
I added another function and change the name of them for:
- getAllRentalsCount -> To get the current index
- getAllRentalsNextCount -> To get the next index